modules/SocketManager/module/inc/SocketManager/socketmanager.h

```diff
@@ -343,6 +343,17 @@ extern indigo_error_t ind_soc_select_and_run(int run_for_ms);
 
 extern void ind_soc_run_status_set(ind_soc_run_status_t status);
 
+/**
+ * Get run status.
+ *
+ * If status is IND_SOC_RUN_STATUS_EXIT then some other modules
+ * may need this status to perform a graceful exit.
+ */
+
+extern void ind_soc_run_status_set(ind_soc_run_status_t status);
```

Comment:
Should this be ind_soc_run_status_get() instead?
And if so, I'm thinking the prototype would be `extern ind_soc_run_status_t ind_soc_run_status_get(void);`